### PR TITLE
topologymanager: Update feature-gates availibilty for TopologyManagerPolicyOptions beta graduation

### PR DESCRIPTION
**Pull request by PiotrProkop**
Enhancement link: https://github.com/kubernetes/enhancements/issues/3545
Implementation link:  https://github.com/kubernetes/kubernetes/pull/118816

### CODE REVIEW

**Comment by PiotrProkop** [comment]
@sftim can you take a look? I missed this part in my 1st PR https://github.com/kubernetes/website/pull/41718

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -201,8 +201,8 @@ For a reference to old feature gates that are removed, please refer to
 | `TopologyAwareHints` | `false` | Beta | 1.23 | 1.23 |
 | `TopologyAwareHints` | `true` | Beta | 1.24 | |
 | `TopologyManagerPolicyAlphaOptions` | `false` | Alpha | 1.26 | |
-| `TopologyManagerPolicyBetaOptions` | `false` | Beta | 1.26 | |
-| `TopologyManagerPolicyOptions` | `false` | Alpha | 1.26 | |
+| `TopologyManagerPolicyBetaOptions` | `true` | Beta | 1.26 | |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:204]:
```suggestion
| `TopologyManagerPolicyBetaOptions` | `true` | Beta | 1.26 | 1. 27 |
```

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks

You also need to change https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-policy-options

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (PiotrProkop) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md:204]: changed

**Comment by PiotrProkop** [comment]
> Thanks
> 
> You also need to change https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-policy-options

It was already changed in https://github.com/kubernetes/website/pull/41718

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (tengqm) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: No. This gate will stay there as Beta for a long time.

**Comment by tengqm** [comment]
/hold cancel
The latest revision is correct.

**Review by sftim — COMMENTED** [review]
Looks right

/lgtm
/approve

**Pull request merged**